core/domain/stats_domain.py

```diff
@@ -1511,15 +1515,6 @@ def __init__(
         self.interaction_id = interaction_id
         self.calculation_output = calculation_output
 
-    def save(self):
-        """Validate the domain object and commit it to storage."""
-        self.validate()
-        stats_models.StateAnswersCalcOutputModel.create_or_update(
```

Comment:
It's weird that we have models in the domain layer, now as this line is removed, can we remove `stats_models` from top?

Reply:
Not yet since the `StateAnswersCalcOutput` still uses it to implement the `save` method.